app/javascript/dashboard/components/helpCenter/Portal/Popover/Index.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <div class="portal-popover__container">
+    <header>
+      <div class="actions">
+        <h2>{{ $t('HELP_CENTER.PORTAL.POPOVER.TITLE') }}</h2>
+        <router-link to="#" class="new-popover-link">
```

Comment:
We can use `a` tag and reuse `button smooth secondary` classes here - `app/javascript/dashboard/components/layout/sidebarComponents/PrimaryNavItem.vue`

Reply:
using the a tag will refresh the page. @nithindavid